src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -2774,6 +2774,68 @@ public static void DCS_TypeWithDelegate()
         Assert.Equal(value.IntProperty, actual.IntProperty);
     }
 
+    [Fact]
+    public static void DCS_ResolveNameReturnsEmptyNamespace()
+    {
+        EmptyNsContainer instance = new EmptyNsContainer(new EmptyNSAddress());
+        var settings = new DataContractSerializerSettings() { MaxItemsInObjectGraph = int.MaxValue, IgnoreExtensionDataObject = false, PreserveObjectReferences = false };
+        EmptyNsContainer result = SerializeAndDeserialize(instance, null, settings, null, true);
```

Comment:
Can you please run the test on desktop and use the actual output from there as the `baseline` for this method? And you would also need to set `skipStringCompare` to `false` to enable the payload comparison.